src/valkey-benchmark.c

```diff
@@ -1541,14 +1549,15 @@ int parseOptions(int argc, char **argv) {
         " -k <boolean>       1=keep alive 0=reconnect (default 1)\n"
         " -r <keyspacelen>   Use random keys for SET/GET/INCR, random values for SADD,\n"
         "                    random members and scores for ZADD.\n"
-        "                    Using this option the benchmark will expand the string\n"
-        "                    __rand_int__ inside an argument with a 12 digits number in\n"
+        "                    Using this option the benchmark will replace the string\n"
+        "                    __rand_int__ inside an argument with a random 12 digit number in\n"
         "                    the specified range from 0 to keyspacelen-1. The\n"
         "                    substitution changes every time a command is executed.\n"
         "                    Default tests use this to hit random keys in the specified\n"
         "                    range.\n"
         "                    Note: If -r is omitted, all commands in a benchmark will\n"
         "                    use the same key.\n"
+        " --sequential       modifies -r argument to replace with sequential 12 digit numbers.\n"
```

Comment:
```suggestion
        " --sequential       Modifies the -r argument to replace the string __rand_int__\n"
        "                    with 12 digit numbers sequentially instead of randomly.\n"
```